src/Compilers/CSharp/Portable/Errors/ErrorCode.cs

```diff
@@ -540,7 +540,9 @@ internal enum ErrorCode
         ERR_PartialMethodInExpressionTree = 765,
         ERR_PartialMethodMustReturnVoid = 766,
         ERR_ExplicitImplCollisionOnRefOut = 767,
-        // unused 768-799
+        ERR_RecursiveConstructorCall2 = 768,
```

Comment:
Sorry I wasn't more clear here. `ERR_IndirectRecursiveConstructorCall` does not currently exist; I was suggesting it as a more clearly-named constant for this particular diagnostic.